bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -848,6 +850,21 @@ protected void runRule(RuntimeRule rule, RuleEngineCallbackImpl.TriggerData td)
 
     }
 
+    private void clearCacheOutputValues(RuntimeRule rule) {
+        List<Trigger> tiggers = rule.getTriggers();
+        for (Trigger trigger : tiggers) {
+            ((RuntimeTrigger) trigger).setOutputs(null);
```

Comment:
I don't think you have to care about triggers and actions here - this is done in https://github.com/marinmitev/smarthome/pull/123/files#diff-d496a3945090abb3b9b346c925d42370R833 already.

You only need to clear the context, so we imho do not even need this method here.